### PR TITLE
chore: create webhooks docs

### PR DESCRIPTION
**Pull request by jmiliMP**

### CODE REVIEW

File: docs/app-payments/webhooks.md [hunk 1]
```diff
@@ -216,4 +216,11 @@ It is possible to provide `paymentPointId` when creating or updating webhooks. T
...
 </tr>
 </tbody>
-</table>
\ No newline at end of file
+</table>
+
+## Create webhook
+You can view and manage webhooks in the [MobilePay portal](https://portal.mobilepay.dk/developer/webhooks) or via [Webhooks Api](/api/wehooks)
+
+When creating an webhook you need to subscribe to events and app payments events are available only if you have App Payments product. The url must be `HTTPS` and unique per webhook.
```
Comment (JonasGavenavicius) [inline docs/app-payments/webhooks.md:224]: a webhook

Comment (JonasGavenavicius) [inline docs/app-payments/webhooks.md:224]: An App Payments

Comment (JonasGavenavicius) [inline docs/app-payments/webhooks.md:224]: URL

Comment (JonasGavenavicius) [inline docs/app-payments/webhooks.md:224]: When creating a webhook, you need to subscribe to events, and app payments events are available only if you have an App Payments product. The URL must be `HTTPS` and unique per webhook.

Comment (JonasGavenavicius) [reply docs/app-payments/webhooks.md:224]: Grammarly approves
image

Comment (jmiliMP) [reply docs/app-payments/webhooks.md]: fixed

**Review by JonasGavenavicius — APPROVED** [review]

**Review by edvinasmuncelis — APPROVED** [review]

**Pull request merged**